CGATPipelines/Pipeline/Execution.py

```diff
@@ -368,8 +368,8 @@ def run(**kwargs):
     options.update(kwargs.items())
 
     # insert a few legacy synonyms
-    options['cluster_options'] = options.get('job_options',
-                                             options['cluster_options'])
+    options['cluster_options'] += " " + options.get('job_options',
+                                                    "")
```

Comment:
Ok, so I have tried to implement that logic here:
https://github.com/CGATOxford/CGATPipelines/pull/254

@IanSudbery could you please test these changes and let us know.

IMHO `job_options` and `cluster_options` should not be combined.